create-testchain-pymaker.sh

```diff
@@ -2,6 +2,7 @@
 
 TAG=$1
 ALLOWED_TAGS="unit-testing integration-testing"
+VERSION=1.1.0
```

Comment:
I believe it's cleaner to hardcode this.  May reconsider later if I think of a way to reduce chance of operator error.

Reply:
that's OK for now, it can be added as a var in travis build too